app/models/consultation.rb

```diff
@@ -371,6 +379,14 @@ def needs_site_visit?
     planning_application.prior_approval? ? neighbour_responses.objection.any? : planning_application.application_type.site_visits?
   end
 
+  def neighbour_review
+    neighbour_reviews.order(:created_at).last
+  end
+
+  def create_neighbour_review!
+    reviews.create!(specific_attributes: {consultation_type: "neighbour"}, status: "complete", assessor: Current.user)
```

Comment:
One reason for keeping the separate associations and not using association extensions is that you can use the `neighbour_reviews` to create the instance and it will set `specific_attributes` for you, e.g.

``` ruby
neighbour_reviews.create!(status: "complete", assessor: Current.user)
```